docs/reference/commandline/cli.md

```diff
@@ -379,7 +377,7 @@ list of root Certificate Authorities.
 
 ## Examples
 
-### <a name="host"></a> Specify daemon host (-H, --host)
+### Specify daemon host (-H, --host) {#host}
```

Comment:
Should we do the same for other commands like https://github.com/docker/cli/blob/master/docs/reference/commandline/build.md?plain=1#L318?

Reply:
> Not sure if cli-docs-tool would need any updating?

Hum I think so otherwise it would not generate link in the markdown table: https://github.com/docker/cli/actions/runs/6197667729/job/16826660657?pr=4566#step:4:35

Reply:
But I think it would break linking in GitHub Flavored Markdown

Reply:
Looks like GFM doesn't support it; https://gist.github.com/asabaylus/3071099?permalink_comment_id=4585378#gistcomment-4585378